core/src/main/java/org/wso2/testgrid/core/command/RunTestPlanCommand.java

```diff
@@ -110,28 +111,30 @@ public void execute() throws CommandExecutionException {
                         testPlanPath.toAbsolutePath()));
             }
 
-            if (log.isDebugEnabled()) {
-                log.debug(
+            if (logger.isDebugEnabled()) {
+                logger.debug(
                         "Input Arguments: \n" +
                         "\tProduct name: " + productName + "\n" +
                         "\tProduct version: " + productVersion + "\n" +
                         "\tChannel" + channel);
-                log.debug("TestPlan contents : \n" + new String(Files.readAllBytes(testPlanPath),
+                logger.debug("TestPlan contents : \n" + new String(Files.readAllBytes(testPlanPath),
                         Charset.forName("UTF-8")));
             }
 
             // Get an infrastructure to execute test plan
             ProductTestPlan productTestPlan = getProductTestPlan(productName, productVersion, channel);
             Optional<String> infraFilePath = getInfraFilePath(productTestPlan);
             if (!infraFilePath.isPresent()) {
-                log.info(StringUtil.concatStrings("No infra files found for the given product test plan - ",
-                        productTestPlan));
+                logger.info(StringUtil.concatStrings("No infra files found for the given product test plan - ",
```

Comment:
`productTestplan.toString()` is not required. We can use productTestplan instance instead.